cmd/simulacra/simulacra.go

```diff
@@ -204,8 +212,43 @@ func getConfigFromYaml(configPath string) (*Config, error) {
 	return &config, nil
 }
 
+func getConfigFromDiagnosticOutput(outputDir string) (*Config, error) {
+	type Metadata struct {
+		Image       string `json:"image"`
+		ProjectName string `json:"project_name"`
+		Zone        string `json:"zone"`
+	}
+
+	metadataFile, err := ioutil.ReadFile(path.Join(outputDir, "vm_config.json"))
+	if err != nil {
+		return nil, err
+	}
+
+	var metadata Metadata
+	if err := json.Unmarshal([]byte(metadataFile), &metadata); err != nil {
+		return nil, err
+	}
+
+	config := &Config{
+		Platform:           metadata.Image,
+		Project:            metadata.ProjectName,
```

Comment:
You definitely won't want this. The `project_name` is going to be the customer's project. This should still be the project where we Simulacra will actually deploy stuff

Reply:
oh this is so true, this is supposed to infer from Gcloud